docs/public/_redirects

```diff
@@ -55,6 +55,7 @@ https://material-ui.dev/* https://material-ui.com/:splat 301!
 /utils/popovers/ /components/popover/ 301
 /utils/* /components/:splat 301
 /api/mui-theme-provider/ /styles/api/ 301
+/components/expansion-panels/ /components/accordion/ 301
```

Comment:
Shouldn't we include `/api` routes as well?

Reply:
good catch